Lesson32/gozde/my-app/src/shared-components/typography/HeadingL/index.jsx

```diff
@@ -0,0 +1,43 @@
+import './styles.css';
```

Comment:
Well done defining all the typography components!
It's more common in React to put each of them in a separate file or folder. In our project structure, create a folder in 'typography' folder for each component. E.g. 
typography/
  HeadingL/
  HeadingM/
  Display/
  HeadingS/
  BodyM/
  BodyS/

Each of these folders will contain index.jsx file (that holds component), e.g. :
```
import './styles.css';

export const HeadingL = ({title, bold}) => {

  return (
    <h2 className={bold ? 'bold' : ''}>{title}</h2>
  );
}
```

and styles.css file that holds styles for this particular component.